lib/util/extensions/context.dart

```diff
@@ -0,0 +1,11 @@
+import 'package:flutter/material.dart';
+import 'package:lemmy_api_client/v3.dart';
+
+import '../../stores/accounts_store.dart';
+import '../observer_consumers.dart';
+
+extension BuildContextExtensions on BuildContext {
+  /// Get default [Jwt] for an instance
+  Jwt? defaultJwt(String instanceHost) =>
```

Comment:
Added it since I noticed we very often do `context.read<AccountsStore>().defaultUserDataFor(instanceHost)?.jwt`. If you don't like it I'll remove it